server/gfmd/auth_uid.c

```diff
@@ -369,15 +369,14 @@ auth_uid_to_global_username_sasl(void *closure,
 		 * to prevent information leak
 		 */
 		gflog_info(GFARM_MSG_UNFIXED,
-		    "unknown user id <%s>", auth_user_id);
+			"unknown user id <%s>, %s", auth_user_id, diag);
```

Comment:
Please use gflog_info(..., "%s: unknown user id <%s>", diag, auth_user_id).
It is against Gfarm practice to write the function name at the end of log line, so please move it to the beginning of the log.